### PR TITLE
Add blog post to weatherforce.github.io

### PR DESCRIPTION
**Pull request by TomTom4**
Hey guys, this is my very first blog post on ipyleaflet-legend. Could you have a look?

### CODE REVIEW

File: content/ipyleaflet-legend_release.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Tags: jupyter, jupyter-notebook, ipyleaflet, ipywidget
+Slug: Ipyleaflet-Legend is released
+Authors: Thomas Pouvreau
+
+
+# Introducing ipyleaflet-legend
+
+It is an implementation on jupyter notebook of a leaflet legend found in this [leaflet tutorial](https://leafletjs.com/examples/choropleth/). More precisely, it refers to the Custom Legend Control section. We won't go too far in details here, just have in mind that it is both a LeafletControl and an ipywidget. So it cannot work outside of ipyleaflet and must be instantiated with it. The good news is that since it is a LeafletControl, it can be positioned like any other ipyleaflet control. 
+
+A second Post will follow or a serie to explain how it was done. It might be an help to everyone seeking for importing leaflet widget on ipyleaflet using ipywidget.
```
Comment (ghost) [inline content/ipyleaflet-legend_release.md]: serie -> series

It might be an help -> It might help

File: content/ipyleaflet-legend_release.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Title: A legend for Ipyleaflet 
+Date: 2020-04-17 12:03
+Category: Python
+Tags: jupyter, jupyter-notebook, ipyleaflet, ipywidget
+Slug: Ipyleaflet-Legend is released
```
Comment (ghost) [inline content/ipyleaflet-legend_release.md]: The slug is the URL path. It shouldn't contain spaces.

File: content/ipyleaflet-legend_release.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+Title: A legend for Ipyleaflet
```
Comment (ghost) [inline content/ipyleaflet-legend_release.md]: ipyleaflet is spelled all lower case, as in the project docs.

**Review by jsoursou — APPROVED** [review]
👍

**Pull request merged**